components/atom/switch/src/index.js

```diff
@@ -16,108 +16,65 @@ export const TYPES = {
   SINGLE: 'single'
 }
 
+const SUPPORTED_KEYS = [' ', 'Enter', 'Spacebar']
+
 class AtomSwitch extends Component {
   state = {
-    toggle: false,
+    isToggle: this.props.initialValue,
```

Comment:
`isToggle` or `isToggled` ?

Reply:
From my perspective, `isToggled` is the correct one. But I followed the `isFocus` structure. I felt I was changing too much. :D